examples/src/main/java/com/example/ftclibexamples/DeadWheelsSample.java

```diff
@@ -34,6 +34,7 @@
 
     public static final double WHEEL_DIAMETER = 2.0;
     public static final double TICKS_PER_REV = 8192;
+    public static final double DISTANCE_PER_PULSE = Math.PI * WHEEL_DIAMETER / TICKS_PER_REV;
```

Comment:
We should probably include a term here for any gearing on the dead wheel.

Reply:
I think it's pretty self explanatory? With TICKS_PER_REV being adjusted for, however that can def be a comment.